Override dict's get method (#505)
cloudant/python-cloudant: CHANGES.md

- [FIXED] Incorrect use of username as account name in `Cloudant.bluemix()`.
- [IMPROVED] Documented use of None account name and url override for `Cloudant.iam()`.
- [FIXED] Use custom encoder (if provided) for all view `key` params not just `keys`.

> - [NEW] Override the `get()` method for CouchDatabase, allowing it to retrive remote docuemnts if the `remote` parameter is specified.

Reviewer: ```suggestion
- [NEW] Override `dict.get` method for `CouchDatabase` to add `remote` parameter allowing it to retrieve a remote document if specified.
```
